components/form/index.tsx

```diff
@@ -141,24 +154,145 @@ export default function StoryForm() {
 
           {/* Anonymous */}
           <Field name="anonymous">
+            {({ field, form }) => {
+              // This little beauty is a result of radio buttons only providing string values, but we really want a boolean
+              // so that we can do some conditional validation (when "Show my name" is selected, we require name)
+              // Copy the  original handler
+              const fieldOnChange = field.onChange
+              // Hijack the value and coerce it to boolean
+              field.onChange = (event) => {
+                if (event?.target?.value) {
+                  const normalizedValue = event.target.value === 'true' ? true : false
+                  event.target.value = normalizedValue
+                }
+                // Call formik's original handler with our replaced event and value
+                fieldOnChange(event)
+              }
+              return (
```

Comment:
I'm baffled that Formik doesn't have a way to transform values. Anyways, this takes "true" or "false" that we get from radios and coerces into `true` or `false`

Reply:
FWIW, I think you can drop the `? true: false` and it'll behave exactly the same. `===` already returns a boolean result, `true` or `false`

Reply:
Also, I think you don't need the optional chaining (`event.target.value` is always present on the event).

Reply:
Ah, yeah - I'll tidy this. The optional chaining (or some kind of undefined check) is required because I was getting `undefined` on re-renders (dunno...) and it was blowing up.